operator-framework-junit5/src/main/java/io/javaoperatorsdk/operator/junit/LocallyRunOperatorExtension.java

```diff
@@ -43,6 +43,8 @@ public class LocallyRunOperatorExtension extends AbstractOperatorExtension {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(LocallyRunOperatorExtension.class);
   private static final int CRD_DELETE_TIMEOUT = 1000;
+  private static final LinkedList<AppliedCRD> appliedCRDs = new LinkedList<>();
+  private static final boolean deleteCRDs = Boolean.parseBoolean(System.getProperty("testsuite.deleteCRDs", "true"));
```

Comment:
Shouldn't be these rather here as other configs?
[Config](https://github.com/operator-framework/java-operator-sdk/blob/7519920c13f10764e355f88adf998c80ff056987/operator-framework-junit5/src/main/java/io/javaoperatorsdk/operator/junit/LocallyRunOperatorExtension.java#L336)

Reply:
I don't think we want to configure this per Test, but rather for the whole testsuite. I think that per test it can be confusing.

Reply:
Hmm, good point,
I'm working this this now:
https://github.com/operator-framework/java-operator-sdk/issues/2688
And here the CRD's will have to be managed manually, but fur sure be at least one step where the CRD is updated. But that should be fine too, so yes, if we see that this canont cover such use cases when in particular tests user wants to manage use cases we can still add a property also to builder.